cypress/tests/integration/Statistics.cy.js

```diff
@@ -28,8 +28,8 @@ describe('Statistics Tests', function() {
 			'Article Details',
 			'articles',
 			['Mwandenga', 'Karbasizaed'],
-			4,
-			2
+			2,
+			1
```

Comment:
The tests should be more independent, so we can run them in parallel and also avoid side-effects, but until we get it done, that's what we can do 😁

Reply:
Were the earlier numbers doubled to accommodate the changes introduced to the submission counts by the import/export tests running first?

Reply:
@asmecher Yes exactly. I moved the import/export to the end as it was breaking Subscription.py in intermittent way. As the same submission existed twice with same title. And since there is mechanism to resolve the submission (on frontend) by the title - it in some cases (by luck and database - it was failing more on psql) pick wrong submission in database.

Reply:
@asmecher resetting database for each integration test to the "after submissions state" would be worthy improvement.

Reply:
Gotcha, thanks! I agree, let's resolve the database reset process as a priority.